c_glib/doc/arrow-glib/arrow-glib-docs.xml

```diff
@@ -91,6 +91,18 @@
     </chapter>
   </part>
 
+  <part id="file-system">
```

Comment:
This ID is duplicated.
How about `file-system-module`, `file-system-api` or something?